addons/l10n_in/views/res_company_views.xml

```diff
@@ -11,9 +11,9 @@
             <xpath expr="//sheet" position="before">
                 <div class="alert alert-warning mt-1 mb-1" role="alert" invisible="not l10n_in_gst_state_warning or country_code != 'IN'">
                     <field name="l10n_in_gst_state_warning"/>
-                    <button name="action_update_state_as_per_gstin"
-                            string="Update it"
-                            class="oe_link"
+                    <a name="action_update_state_as_per_gstin"
```

Comment:
Why can not it be done with button oe_link?

Reply:
The button has a margin that misaligns it with the rest of the warning, making it look unappealing. That's why I used an <a>.

Reply:
@jco-odoo